src/test/functional/supportedBrowsers.js

```diff
@@ -0,0 +1,65 @@
+const supportedBrowsers = {
+  // edge: {
```

Comment:
are we not doing edge testing Sunday?

Reply:
its not working at this point in time. the plan is to get it working at some point in future